src/array.ts

```diff
@@ -83,17 +84,23 @@ export function from<T>(arrayLike: (string | ArrayLike<T>), mapFunction?: MapCal
 	}
 
 	const Constructor: any = this;
```

Comment:
It's not entirely clear to me why `Constructor` is set to `this`, since `this` does not actually represent a sub-classed array. Does this assume that `from` is executed as `from.call(SubclassedArray, arrayLike)`?